userscript/origin.js

```diff
@@ -0,0 +1,26 @@
+// ==UserScript==
+// @name         OSRT Origin
+// @namespace    openSUSE/openSUSE-release-tools
+// @version      0.1.0
+// @description  Supplement OBS interface with origin information.
+// @author       Jimmy Berry
+// @match        */package/show/*
+// @require      https://code.jquery.com/jquery-3.3.1.min.js
+// @grant        none
+// ==/UserScript==
+
+(function()
+{
+    var pathParts = window.location.pathname.split('/');
+    var project = pathParts[pathParts.length - 2];
+    var package = pathParts[pathParts.length - 1];
+
+    var domain_parent = window.location.hostname.split('.').splice(1).join('.');
+    var subdomain = domain_parent.endsWith('suse.de') ? 'tortuga' : 'operator';
```

Comment:
please don't abuse tortuga for that. Tortuga is the openqa installation used by 3+ developers 

You have 2 openstack installations available to you